backend/prisma/migrations/20231105081703_change_model_name_to_singular/migration.sql

```diff
@@ -0,0 +1,59 @@
+/*
+  Warnings:
+
+  - You are about to drop the `publications` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `publications_traveler` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `travelers` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "publications_traveler" DROP CONSTRAINT "publications_traveler_publication_id_fkey";
+
+-- DropForeignKey
+ALTER TABLE "publications_traveler" DROP CONSTRAINT "publications_traveler_traveler_id_fkey";
+
+-- DropTable
+DROP TABLE "publications";
+
+-- DropTable
+DROP TABLE "publications_traveler";
+
+-- DropTable
+DROP TABLE "travelers";
+
+-- CreateTable
+CREATE TABLE "traveler" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
```

Comment:
I might be wrong but I think the quotes on the column names aren't necessary? Looks like you used consistent quoting though so up to your styling preference!